doc/modules/feature_selection.rst

```diff
@@ -234,11 +234,27 @@ Randomized sparse models
 
 .. currentmodule:: sklearn.linear_model
 
-The limitation of L1-based sparse models is that faced with a group of
-very correlated features, they will select only one. To mitigate this
-problem, it is possible to use randomization techniques, reestimating the
-sparse model many times perturbing the design matrix or sub-sampling data
-and counting how many times a given regressor is selected.
+In terms of feature selection, there are some well-known limitations of
+L1-penalized models for regression and classification.  For example, it is
+known that the Lasso will tend to select an individual variable out of a group
+of highly correlated features.  Furthermore, even when the correlation between
+features is not too high, the conditions under which L1-penalized methods
+consistently select "good" features can be restrictive in general.
+
+To mitigate this problem, it is possible to use a randomization technique
+known as stability selection.  In this method, a subsample of the data is fit
+to a L1-penalized model where the penalty of a random subset of coefficients
+has been scaled.  Specifically, given a subsample of the data :math:`(y_i,
+x_i), i \in I`, where :math:`I \subset \{1, 2, \ldots, n\}` is a random subset
+of the data of size :math:`n_I`, the following modified Lasso fit is obtained:
+
+.. math::   \hat{w_I} = \mathrm{arg}\min_{w} \frac{1}{2n_I} \sum_{i \in I} (y_i - x_i^T w)^2 + \alpha \sum_{j=1}^p \frac{|w_j|}{s_j},
+
+where :math:`s_j \in \{s, 1\}` are independent trials of a fair Bernoulli
```

Comment:
Yup, agree with you. Staying consistent internally is better to match the paper's notation. Though, there is an additional "issue". This is that `alpha` and `C` in the code refer to the penalization in the Lasso and Logistic regression models, respectively. And in the code, `alpha` is not used as scaler, but a variable called `weights`. I just submitted a change that does not call `alpha` the scaling parameter, but just `s`. This might be better. What do you think?